scaling_performance/scaling_cluster_monitoring_operator.adoc

```diff
@@ -0,0 +1,152 @@
+[[scaling-performance-cluster-metrics]]
+= Scaling Cluster Metrics
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+link:https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator]. As an
+{product-title} administrator, you can view system resources, containers and components metrics in
+one dashboard interface.
+
+This topic provides information for scaling the metrics components.
+
+ifdef::openshift-enterprise[]
+[[cluster-metrics-horizontal-pod-autoscaling]]
+[NOTE]
+====
+Autoscaling the metrics components.
+====
+endif::[]
+
+
+[[metrics-recommendations-for-OCP]]
+== Recommendations for {product-title}
+
+* Run metrics pods on dedicated {product-title}
+xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure
+nodes].
+* Use persistent storage when configuring metrics. Set
+`USE_PERSISTENT_STORAGE=true`.
+* use GlusterFS as storage on top of non-volatile memory express (NVMe) drives link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs[*openshift_storage_glusterfs*].
+
+[[scaling-performance-capacity-planning]]
+== Capacity Planning for Cluster Metrics
+
+A various tests were performed for different scale sizes, we've seen the the Prometheus database grew at the
```

Comment:
Various tests were performed for different scale sizes. The Prometheus database grew, as reflected in the table below.